platforms/python/buildPython.sh

```diff
@@ -18,11 +18,20 @@ mkdir -p "$targetDir"
 
 builtPythonPrereqs=false
 buildPythonPrereqsImage() {
+    debianType=$debianFlavor
+	# stretch is out of support for python, but we still need to build stretch based
+	# binaries because of static sites, they need to move to buster based jamstack image
+	# before we can remove this hack
```

Comment:
@arroyc may you elaborate more on this? Or link to the action item please

Reply:
https://devdiv.visualstudio.com/DevDiv/_boards/board/t/Oryx/Stories/?workitem=1428737